src/components/ActionsModal/components/ContractsList/ContractsList.tsx

```diff
@@ -61,7 +61,7 @@ const ContractsList: React.FC<ContractsListProps> = ({
             Set Permissions
           </ButtonLabel>
         </ActionsButton>
-        {!isRepGuild ? (
+        {!!isRepGuild ? (

```

Comment:
This was wrong all this time?

Reply:
Hahahahaha, yes somehow. It wasnt an issue previously because we didn't have matching bytecodes for goerli and weren't really testing REP guilds as much.

Reply:
👀